### PR TITLE
Docs: Iconography guidelines

### PR DESCRIPTION
**Pull request by ponori**
Added Iconography guidelines to the docs

### CODE REVIEW

File: docs/pages/iconography.js [hunk 1]
```diff
@@ -0,0 +1,402 @@
...
+                  nationality or language.
+                </Text>
+              }
+            />
+          </Flex.Item>
+        </Flex>
+      </MainSection>
+
+      <MainSection
+        name="Typography components"
```
Comment (ayeshakmaz) [inline docs/pages/iconography.js:106]: Should this be "Iconography components" or "Components with Icons" maybe?

File: docs/pages/iconography.js [hunk 2]
```diff
@@ -0,0 +1,402 @@
...
+
+- [IconButton](/icon): Uses icons instead of text to convey available actions on a screen.
+- [Status](/status): A graphic indicator of an element’s state.
+- [Link](/link): External links use the visit icon as an external domain indicator.
+`}
+      />
+
+      <MainSection name="Characteristics">
+        <MainSection.Subsection title="Appearance">
+          <Markdown
```
Comment (ayeshakmaz) [inline docs/pages/iconography.js:118]: I don't remember so apologies if we've discussed this, but can this content go in the "description" prop of Subsection instead? It should still be rendered in markdown

File: docs/pages/iconography.js (hunk 1 above)
Comment (ponori) [reply docs/pages/iconography.js]: Thanks! Fixed!

File: docs/pages/iconography.js (hunk 2 above)
Comment (ponori) [reply docs/pages/iconography.js:118]: I added it this way as the content is intermixed w/ an image so it felt strange to have one block in a `description` prop and then another as a `Markdown` component. Thoughts?

**Review by ayeshakmaz — APPROVED** [review]

**Pull request merged**